bin/ncm-ci.js

```diff
@@ -7,6 +7,12 @@ const graphql = require('../lib/graphql')
 const chalk = require('chalk')
 const fetch = require('node-fetch')
 
+// Setting up a timeout to be used with fetch() – will use the NCM_TIMEOUT environment
+// variable or default to 2 minutes (120 seconds / 120000 milliseconds)
+const timeout = process.env.NCM_TIMEOUT
+  ? Number(process.env.NCM_TIMEOUT) * 1000
```

Comment:
still need to check for a non-numeric string in the env var:

```console
$ node -p 'Number("foo") * 1000'
NaN
```

`timeout` will end up being a `NaN` in that case